code/nodes/saveGraph/utils.q

```diff
@@ -1,13 +1,16 @@
-\d .automl
+// code/nodes/saveGraph/utils.q - Utilities for the saveGraph node
+// Copyright (c) 2021 Kx Systems Inc
+//
+// Utility functions specific the the saveGraph node implementation
 
-// Utility functions for saveGraph
+\d .automl
 
 // @kind function
 // @category saveGraphUtility
-// @fileoverview Create regression target distribution plot and save down locally
-// @param params    {dict} All data generated during the process
-// @param savePath  {str} Path to where plots are to be saved
-// @return {null} Target distribution plot saved to appropriate location
+// @desc Create regression target distribution plot and save down locally
+// @param params    {dictionary} All data generated during the process
```

Comment:
Unnecessary spacing between param name and param type can be removed for the rest of this file